### ISSUE TITLE
Revisit need for node/electron-browser in workbench/contrib/experiments

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#69108]
Refs https://github.com/Microsoft/vscode/issues/68302

I am not sure why `src/vs/workbench/contrib/experiments` has to have most of its files in `electron-browser` land. Please consider lifting this to `browser` if possible.

If there is something blocking this task, speak up and we need to think about introducing new services as needed.

**Comment by bpasero** [comment]
@sandy081 adding you because the extension viewlet seems to be a consumer of the `IExperimentService` and please note that currently there does not seem to exist a "simple" version of it for the web. 

Ideally we could:
* move `IExperimentService` to `electron-browser` to signal it is only available there
* break up the dependency to the extensions viewlet if possible

Or:
* make `IExperimentService` available in the web, by breaking up the dependency to `IWorkspaceStatsService` (that seems to drag it)

**Comment by sandy081** [comment]
@bpasero There is simple version of `IExperimentService` which is needed for extensions viewlet to be rendered in web.

I made following changes

1. Moved `IWorkspaceStatsService` to common
2. Introduced simple service for `IWorkspaceStatsService` for web
3. With this moved `IExperimentService` completely to web
4. Removed simple service for `IExperimentService` in web

with this web runs the real `IExperimentService` with simple `IWorkspaceStatsService`. I will create a follow up task for @RMacfarlane to look into workspace stats in web.
